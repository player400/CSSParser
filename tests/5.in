#breadcrumb 
{
	width: 80%;
	font-family: Verdana;
	font-size: 9pt;
}

body {
	min-width: 780px;
	margin: 0;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

h1, h2, h3, h4, h5, h6 {color: #0066CB;}

h1 {
	font-size: 2.0em;
	font-weight: normal;
}

body p{
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

body div {
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		



#mainwrapper {			
	margin: 0 auto;
	width: 80%;
	min-width: 780px;
	text-align: left;
}

#header {
	margin-left: 0px;
	margin-bottom: 0px;
	color: #FFFFFF;
	background-color: #8CA7BB;	
	border:solid 1px #005AC5;
	font-size: 2.0em;
	font-weight: normal;
	padding:10px;
	border-bottom:solid 4px Black;
}

#header-bottom
{
	width:100%;
	font-size: 14px;
	font-style: italic;
}

#content {
	width: 100%;
	border:solid 1px #005AC5;
	background-color:White;
	border-top:solid 1px Black;
}

#content-left {
	width:180px;
	padding-top:10px;
	}

#content-right 
{
	width:auto;
	border-left:1px solid #A7A7A7;
	padding:15px;
	vertical-align:top;
}

#breadcrumb 
{
	width: 100%;
	font-family: Verdana;
	font-size: 9pt;
}

#breadcrumb a 
{
	color: #0066CB;
	text-decoration: none;
}

#breadcrumb a:hover 
{
	text-decoration: underline;
}

#footer
{
	width:100%;
	font-size: 0.8em;
	font-style: italic;
	text-align: left;
}
	
.menu_nodeStyle
{
	color:Black;
	font-family:"Verdana";
	font-size:8pt;
	padding: 5px 5px 5px 5px;
	width:100%;
	border:solid 1px White;
}

.menu_hoverNodestyle
{
	background-color:#CCCCCC;
	border:solid 1px #888888;
	cursor:pointer;
}

.menu_selectedNodestyle
{
	padding: 5px 5px 5px 5px;
	font-weight:bold;
	color:#0066CB;
}

????
?
1,S,?
1,A,?
2,S,?
3,A,?
1,S,1
1,A,padding
2,S,2
font-size,A,?
color,A,?
h1,S,?
#breadcrumb,E,width
****
.menu_hoverNodestyle
{
	background-color:#CCCCCC;
	border:solid 1px #888888;
	cursor:pointer;
}
h1, .menu_selectedNodestyle
{
	padding: 5px 5px 5px 5px;
	font-weight:bold;
	color:#0066CB;
}

#breadcrumb 
{
	width: 70%;
	font-family: Verdana;
	font-size: 9pt;
}

.menu_nodeStyle
{
	color:Black;
	font-family:"Verdana";
	font-size:8pt;
	padding: 5px 5px 5px 5px;
	width:100%;
	border:solid 1px White;
}

body p{
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B0CFE6;
}		

body div {
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B1CFE6;
	min-width: 880px;
	margin: 30;
	padding: 10;
	border: 20;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		



????
?
24,A,?
25,A,?
font-size,A,?
h1,S,?
#breadcrumb,E,width
24,A,min-width
24,A,margin
24,A,padding
24,A,border
24,A,background-color
25,A,min-width
25,A,margin
25,A,padding
25,A,border
25,A,background-color

****
next_body p{
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B0CFE6;
}		

next_body div {
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B1CFE6;
	min-width: 880px;
	margin: 30;
	padding: 10;
	border: 20;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		
????
?
24,A,?
25,A,?
24,S,?
25,S,?
font-size,A,?
h1,S,?

****

#breadcrumb 
{
	width: 80%;
	font-family: Verdana;
	font-size: 9pt;
}

body {
	min-width: 780px;
	margin: 0;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

h1, h2, h3, h4, h5, h6 {color: #0066CB;}

h1 {
	font-size: 2.0em;
	font-weight: normal;
}

body p{
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

body div {
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		



#mainwrapper {			
	margin: 0 auto;
	width: 80%;
	min-width: 780px;
	text-align: left;
}

#header {
	margin-left: 0px;
	margin-bottom: 0px;
	color: #FFFFFF;
	background-color: #8CA7BB;	
	border:solid 1px #005AC5;
	font-size: 2.0em;
	font-weight: normal;
	padding:10px;
	border-bottom:solid 4px Black;
}

#header-bottom
{
	width:100%;
	font-size: 14px;
	font-style: italic;
}

#content {
	width: 100%;
	border:solid 1px #005AC5;
	background-color:White;
	border-top:solid 1px Black;
}

#content-left {
	width:180px;
	padding-top:10px;
	}

#content-right 
{
	width:auto;
	border-left:1px solid #A7A7A7;
	padding:15px;
	vertical-align:top;
}

#breadcrumb 
{
	width: 100%;
	font-family: Verdana;
	font-size: 9pt;
}

#breadcrumb a 
{
	color: #0066CB;
	text-decoration: none;
}

#breadcrumb a:hover 
{
	text-decoration: underline;
}

#footer
{
	width:100%;
	font-size: 0.8em;
	font-style: italic;
	text-align: left;
}
	
.menu_nodeStyle
{
	color:Black;
	font-family:"Verdana";
	font-size:8pt;
	padding: 5px 5px 5px 5px;
	width:100%;
	border:solid 1px White;
}

.menu_hoverNodestyle
{
	background-color:#CCCCCC;
	border:solid 1px #888888;
	cursor:pointer;
}

.menu_selectedNodestyle
{
	padding: 5px 5px 5px 5px;
	font-weight:bold;
	color:#0066CB;
}

#breadcrumb 
{
	width: 80%;
	font-family: Verdana;
	font-size: 9pt;
}

body {
	min-width: 780px;
	margin: 0;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

h1, h2, h3, h4, h5, h6 {color: #0066CB;}

h1 {
	font-size: 2.0em;
	font-weight: normal;
}

body p{
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		

body div {
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
	min-width: 780px;
	margin: 20;
	padding: 0;
	border: 10;
	text-align: center;
	font-family: "Trebuchet MS", "Lucida Grande", Arial, Helvetica, sans-serif;
	font-size: 85%;
	color: #666666;
	background-color: #B2CFE6;
}		



#mainwrapper {			
	margin: 0 auto;
	width: 80%;
	min-width: 780px;
	text-align: left;
}

#header {
	margin-left: 0px;
	margin-bottom: 0px;
	color: #FFFFFF;
	background-color: #8CA7BB;	
	border:solid 1px #005AC5;
	font-size: 2.0em;
	font-weight: normal;
	padding:10px;
	border-bottom:solid 4px Black;
}

#header-bottom
{
	width:100%;
	font-size: 14px;
	font-style: italic;
}

#content {
	width: 100%;
	border:solid 1px #005AC5;
	background-color:White;
	border-top:solid 1px Black;
}

#content-left {
	width:180px;
	padding-top:10px;
	}

#content-right 
{
	width:auto;
	border-left:1px solid #A7A7A7;
	padding:15px;
	vertical-align:top;
}

#breadcrumb 
{
	width: 100%;
	font-family: Verdana;
	font-size: 9pt;
}

#breadcrumb a 
{
	color: #0066CB;
	text-decoration: none;
}

#breadcrumb a:hover 
{
	text-decoration: underline;
}

#footer
{
	width:100%;
	font-size: 0.8em;
	font-style: italic;
	text-align: left;
}
	
.menu_nodeStyle
{
	color:Black;
	font-family:"Verdana";
	font-size:8pt;
	padding: 5px 5px 5px 5px;
	width:100%;
	border:solid 1px White;
}

.menu_hoverNodestyle
{
	background-color:#CCCCCC;
	border:solid 1px #888888;
	cursor:pointer;
}

.menu_selectedNodestyle
{
	padding: 5px 5px 5px 5px;
	font-weight:bold;
	color:#0066CB;
}

.menu_nodeStyle
{
	color:Black;
	font-family:"Verdana";
	font-size:8pt;
	padding: 5px 5px 5px 5px;
	width:100%;
	border:solid 1px White;
}

.menu_hoverNodestyle
{
	background-color:#CCCCCC;
	border:solid 1px #888888;
	cursor:pointer;
}

.menu_selectedNodestyle
{
	padding: 5px 5px 5px 5px;
	font-weight:bold;
	color:#0066CB;
}

????
24,A,?
25,A,?
24,S,?
25,S,?
font-size,A,?
h1,S,?
1,S,?
1,A,?
2,S,?
3,A,?
1,S,1
1,A,padding
2,S,2
font-size,A,?
color,A,?
h1,S,?
#breadcrumb,E,width
?
