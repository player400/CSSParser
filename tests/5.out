? == 19
1,S,? == 1
1,A,? == 3
2,S,? == 1
3,A,? == 1
1,S,1 == #breadcrumb
font-size,A,? == 10
color,A,? == 8
h1,S,? == 2
#breadcrumb,E,width == 100%
? == 25
24,A,? == 9
25,A,? == 9
font-size,A,? == 14
h1,S,? == 3
#breadcrumb,E,width == 70%
24,A,min-width == 780px
24,A,margin == 20
24,A,padding == 0
24,A,border == 10
24,A,background-color == #B0CFE6
25,A,min-width == 880px
25,A,margin == 30
25,A,padding == 10
25,A,border == 20
25,A,background-color == #B2CFE6
? == 27
24,A,? == 9
25,A,? == 9
24,S,? == 1
25,S,? == 1
font-size,A,? == 16
h1,S,? == 3
24,A,? == 9
25,A,? == 9
24,S,? == 1
25,S,? == 1
font-size,A,? == 37
h1,S,? == 7
1,S,? == 1
1,A,? == 3
2,S,? == 1
3,A,? == 1
1,S,1 == #breadcrumb
font-size,A,? == 37
color,A,? == 32
h1,S,? == 7
#breadcrumb,E,width == 100%
? == 68
